zebra-consensus/src/chain.rs

```diff
@@ -132,7 +148,10 @@ where
     tracing::info!(?tip, ?max_checkpoint_height, "initializing chain verifier");
 
     let block = BlockVerifier::new(network, state_service.clone());
-    let checkpoint = CheckpointVerifier::from_checkpoint_list(list, tip, state_service);
+    let checkpoint = Buffer::new(
+        CheckpointVerifier::from_checkpoint_list(list, tip, state_service),
+        3,
```

Comment:
Please explain why you chose this bound for the buffer.

I think we might eventually need a bound of 4:
* sync service
* inbound service
* mining service (a feature we might add in future)
* 1 extra slot to avoid contention

We're going to use this bound for the block verifier and checkpoint verifier buffer. So let's turn it into a constant, and put an explanation in a doc comment on the constant.

Reply:
(Extra slots are pretty harmless, but missing slots can slow Zebra down a lot.)